src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Binary;
+using Sparrow.Server;
+using Sparrow.Server.Utils;
+using Voron;
+using Voron.Data.Tables;
+using Voron.Impl;
+
+namespace Raven.Server.Documents.TimeSeries
+{
+    public class TimeSeriesStats
+    {
+        private static readonly Slice TimeSeriesStatsKey;
+        private static readonly Slice PolicyIndex;
+        private static readonly Slice StartTimeIndex;
+        private static readonly TableSchema TimeSeriesStatsSchema = new TableSchema();
+
+        private enum StatsColumn
+        {
+            Key = 0, // documentId, separator, name
+            PolicyName = 1,
+            Separator = 2,
```

Comment:
Why is this its own dedicated column?

Reply:
Easier to work with, instead of adding/removing the separator each time.
Is it a problem?